spec/requests/das_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe 'DAS' do
+  let!(:user) { create(:user, id: 23, name: 'Britney Spears') }
+  let!(:genotype) { create(:genotype, id: 42, user: user) }
+  let!(:snp) { create(:snp, name: 'rs123', chromosome: 7, position: 24926827) }
+  let!(:user_snp) { UserSnp.new(snp, genotype, 'AC').save }
+
+  it 'returns DAS data' do
+    get '/das/23/features', { segment: '7:0,24926827', type: 'AC' },
+                            { 'server-software' => 'Foo 1.0' }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.